sd-2019-stack-overflow-frontend/src/view/AnswerDetails.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const AnswerDetails = ( {answer, onUpvoteAnswer, onDownvoteAnswer, onEditAnswer} ) => (
+    <article className="media" key={answer.id}>
+        <figure className="media-left">
+            <div>
+                <p className="heading">Vote score</p>
+                <p className="title">{answer.voteScore}</p>
+            </div>
+        </figure>
+        <div className="media-content">
+            <div className="content">
+                <p>
+                    {answer.text}
+                    <br />
+                    <br />
+                    {
+                        // eslint-disable-next-line
```

Comment:
:) It is fine to do this now. In the future though, you have to keep in mind to prefer using buttons instead of anchors for actions which don't have a href.